mas-foundation/src/main/java/com/ca/mas/core/conf/ConfigurationManager.java

```diff
@@ -258,6 +267,12 @@ public ConfigurationProvider create(JSONObject jsonObject) throws JSONException
                 conf.setAlsoTrustPublicPki((Boolean) getValue(Config.TRUSTED_PUBLIC_PKI, jsonObject, Boolean.FALSE));
                 continue;
             }
+
+            if (attr == Config.SSL_PINNING_ENABLED) {
```

Comment:
combine this in the previous if condition using ||

Reply:
Updated the method call for this if condition.